lib/start-server.js

```diff
@@ -68,7 +71,7 @@ function checkPort() {
       } else {
         // start local server on specified port
         const server = require('./server/server.js');
-        server(port);
+        server(port, program);
```

Comment:
Would be safer to just pass the options into server: `server(port, program.opts())`.